dNet/dServer.cpp

```diff
@@ -75,7 +75,9 @@ dServer::dServer(const std::string& ip, int port, int instanceID, int maxConnect
 	//Connect to master if we are not master:
 	if (serverType != ServerType::Master) {
 		SetupForMasterConnection();
-		ConnectToMaster();
+		if (!ConnectToMaster()) {
```

Comment:
I believe this log will be a bit misleading; if I recall correctly, RakNet returning true from connect does not mean you are connected.  Is there a different way to phrase this possible failure to connect ti master?